lib/web_ui/lib/src/engine/semantics/checkable.dart

```diff
@@ -28,7 +27,8 @@ enum _CheckableKind {
   toggle,
 }
 
-_CheckableKind _checkableKindFromSemanticsFlag(SemanticsObject semanticsObject) {
+_CheckableKind _checkableKindFromSemanticsFlag(
+    SemanticsObject semanticsObject) {
```

Comment:
nit: undo this formatting change?

Reply:
we use dartfmt in the web engine code instead of arranging indentations manually.